client/src/pages/Initial/LandingPage<Sherry>/LandingPage<Sherry>.scss

```diff
@@ -0,0 +1,37 @@
+* {
```

Comment:
when you're working on any styles, make sure to use very unique classnames, for example `landing-page-sz-top`.
in react, all the classnames and their style are shared with all components, so by making very unique classnames you'll be able to ensure that the style you have is not going to affect any components.

additionally, you can take a look at `App.scss`, this is where we define the style for "global" components like `*`, `:root`, `body`. you'll need to rename these in your component and use `className` to define these styles instead.